src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
Please refer to https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery common mistakes document. Hint: mistake in method signature